tests/test_server.py

```diff
@@ -119,6 +119,20 @@ def test_wsgi_app_dispatch(self):
         server(environ, mock_start_response)
         mock_registry.dispatch.assert_called_once_with(mock.ANY)
 
+    def test_wsgi_request_to_root_endpoint(self):
```

Comment:
This test should also check that the response you get is what is expected. Right now it's basically testing that we do not raise an exception when we pass in a message to the root path.